### PR TITLE
Test add and remove methods

### PR DESCRIPTION
**Pull request by karelvanoordt**
- Used jest to test add and remove methods for To Do List app.
- Mocked DOM to simulate html.

### CODE REVIEW

File: src/__tests__/createTask.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import { jsxEmptyExpression } from '@babel/types'; // eslint-disable-line
```
Comment (helman101) [inline src/__tests__/createTask.js:1]: - [ ] This file should be called `[File_name].test.js` in order to work properly, please change this. 😄

File: src/__tests__/createTask.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+import tasks from '../__mocks__/createTask.js';
+
+const jsdom = require('jsdom');
+
+const { JSDOM } = jsdom;
+
+const dom = new JSDOM(`<!DOCTYPE html><div id='main'></div>`); // eslint-disable-line
+const document = dom.window.document; // eslint-disable-line
+
+describe('Test add method', () => {
```
Comment (helman101) [inline src/__tests__/createTask.js:13]: - [ ] You have only tested the `addTask` and `removeTask` funtionality, please remember that you have to test the majority of the [pure funtions](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwiAtriui__zAhW7h_0HHQcLD5wQFnoECA8QAw&url=https%3A%2F%2Fwww.sitepoint.com%2Ffunctional-programming-pure-functions%2F&usg=AOvVaw06n7VkqKrBdswbXrmU9T9H) 🚀

**Review by helman101 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 🔴 

Hi @karelvanoordt & @Mesi21   👋, 

Nice job so far with this project 😄, but there are still some changes to do.

## Required Changes ♻️

- [ ] _Check the comments under the review._

Cheers and Happy coding!👏👏👏

gif

If anything is not 100% clear please leave your questions or comments in the PR thread, we will be glad to help you 😄.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/__tests__/createTask.js (hunk 2 above)
Comment (karelvanoordt) [reply src/__tests__/createTask.js:13]: Hi Andres, thank you for your review.

Just one comment. This review is for the first project submission (https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/tests_1.md)
And Here it says this:
 "Create a test file ([..].test.js) for a file containing the **add item** and **delete item** functions that you want to test."

I think for the next part of this week's project we need to test the other functions as well.

**Review by carlos-ssh — APPROVED** [review]
# 🍾🍾 **STATUS APPROVED** 👏🏽👏🏽

Hi @Mesi21 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Approved


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**